Add reference to pointer-events property (#29637)
mdn/content: files/en-us/web/css/opacity/index.md

Using `opacity` with a value other than `1` places the element in a new [stacking context](/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context).

> When `opacity` value is set to `0`, the element and all of its children are not visible; however, they still register [pointer events](/en-US/docs/Web/API/Pointer_events). This behavior can be controlled by [pointer-events](/en-US/docs/Web/CSS/pointer-events) property.

Reviewer: ```suggestion
When `opacity` value is set to `0`, the element and all of its children are not visible; however, they still register [pointer events](/en-US/docs/Web/API/Pointer_events). This can be controlled with the CSS [`pointer-events`](/en-US/docs/Web/CSS/pointer-events) property.
```
